qemu/tests/nbd_unix_connection.py

```diff
@@ -0,0 +1,56 @@
+import logging
+
+from virttest import data_dir
+from virttest import storage
+from virttest import qemu_storage
+from virttest import error_context
+
+from avocado.utils import process
+
+from provider.nbd_image_export import QemuNBDExportImage
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    1) Clone the system image1 with qemu-img
+    2) Export the cloned image with qemu-nbd(type=unix)
+    3) Stop to export nbd image
+    4) Repeat connect nbd image by nbdsh cmd
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    def _prepare():
```

Comment:
clone might not be required for your test, which also costs long time, just create an image then export it, or export 'image1', i.e. the vm system image, your testing should not do writes into the exported image, so it's safe

Reply:
Thanks. I choose to create a new image in this case, and the image will be removed and stopped exporting after the test run.